src/utils/address.js

```diff
@@ -0,0 +1,30 @@
+import { validateName } from './utils'
+
+export const ETH_ADDRESS_TYPE = {
+  name: 'name',
+  address: 'address',
+  error: 'error'
+}
+
+export function getEthAddressType(address) {
+  if (!address) return ETH_ADDRESS_TYPE.error
+
+  if (
+    address.length === 42 &&
```

Comment:
Can we not use this? https://web3js.readthedocs.io/en/1.0/web3-utils.html#isaddress

Reply:
We can probably use a utils from ethers. We're not using web3.js library anymore